Dockerfile

```diff
@@ -1,24 +1,28 @@
-FROM node:14
+FROM node:14-alpine@sha256:5c33bc6f021453ae2e393e6e20650a4df0a4737b1882d389f17069dc1933fdc5
 
-RUN mkdir /public && \
-    addgroup --system nodejs --gid 998 && \
+USER root
```

Comment:
I thought containers shouldn't run as a root user? 
https://medium.com/@mccode/processes-in-containers-should-not-run-as-root-2feae3f0df3b

Reply:
@sulthan-ahmed look at the rest of the Dockerfile. You need to start as root to setup a new user and user group with limited permissions. It then switches to that user

Reply:
ahh ok, I didn't know you are allowed to do that.  It seems like it's fairly normal behaviour, thanks for teaching me something 🙂